warehouse/models/staging/rt/stg_rt__feed_fetch_errors.sql

```diff
@@ -0,0 +1,85 @@
+{{
+    config(
+        materialized='incremental',
+        unique_key='event_id',
+        partitions=['current_timestamp()'],
```

Comment:
I don't think partitions will do anything here since you are using the default `merge` strategy

Reply:
Although I'd probably actually use `insert_overwrite` here anyways, since the data is partitioned by `dt` first and you can generally assume you want to update some X days of data at a time

Reply:
Resolved by [5d33338](https://github.com/cal-itp/data-infra/pull/2345/commits/5d333383e4f601bb485fb696e636f5131bcc7fc3)